airbyte-workers/src/main/java/io/airbyte/workers/utils/ConfigReplacer.java

```diff
@@ -89,6 +92,8 @@ public AllowedHosts getAllowedHosts(AllowedHosts allowedHosts, JsonNode config)
           "All allowedHosts values are un-replaced.  Check this connector's configuration or actor definition - " + allowedHosts.getHosts());
```

Comment:
What does it mean to be unreplaced? Is it that we just counldnt parse it and thus likely dont know how to interpret it at the firewall level?

Reply:
I'm worried about drift between the allowedHosts (`${host}`) and the config/spec of the connector (used to be `host` but was recently moved to `hostname`).  In that world, the sync would fail because access to the new `hostname` would be prevented.  At least this log message gives us a pointed that none of the allowedHost interpolations succeeded... maybe check that.